nncf/common/utils/logger/__init__.py

```diff
@@ -0,0 +1,26 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+#pylint:disable=wrong-import-position
+from nncf.common.logging.logger import warning_deprecated
+
+warning_deprecated(
```

Comment:
@MaximProshin, can we do such a deprecation considering that this is the LTS release of OpenVINO and, in fact, this rule should be spread to all its tools?

Reply:
There are no deprecation rules concerning LTS, only the recommendation that HW support should not be deprecated in LTS. Moreover, recent example with Python 3.6 shows that even removal is allowed. Anyway, I would say we should rely on whether a deprecation is really needed rather than on some rules. I personally don't have a strong opinion here. @vshampor , how is it important to state that this is deprecated? Can we just say it's not recommended?

Reply:
If there is no problem with deprecation I would keep "deprecated" in the message because this is a common approach. No comments from my side.